lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -239,10 +282,22 @@ def error_code_from(action, response)
         code&.to_s
       end
 
+      def error_code_from_get(response)
+        return if success_from_get(response)
+        if !response.instance_of? Array
+          code = response['status_code'] || response['code']
+          code&.to_s
+        end
+      end
+
       def url(action, parameters, options = {})
         "#{(test? ? test_url : live_url)}/#{endpoint(action, parameters, options)}/"
       end
 
+      def get_url(action, parameters={}, options={})
+        "#{(test? ? test_url : live_url)}/#{endpoint(action, parameters, options)}"
+      end
+
```

Comment:
This seems to be redundant. I see the only difference to be the trailing `/`. Can we remove that in the `url()`? I don't think it would affect the existing client calls.

Reply:
Removed all tests are passing.